.github/workflows/ci.yml

```diff
@@ -35,7 +36,7 @@ jobs:
         run: forge build
         working-directory: ./ethereum
       - name: Build
-        run: cargo build --release --all-targets
+        run: cargo build --package vft-manager --release --all-targets && cargo build --release --all-targets
```

Comment:
Please check if https://github.com/gear-tech/gear-bridges/actions/runs/12136399913 is also broken